libhoney.go

```diff
@@ -235,6 +261,7 @@ type dynamicField struct {
 //
 // Make sure to call Close() to flush buffers.
 func Init(config Config) error {
+	rand.Seed(time.Now().UnixNano())
```

Comment:
I believe go's rand is deterministic unless given an appropriate seed.  Nothing to do with the rest of this diff, just along for the ride. Just discovered, buried in the package docs, "If Seed is not called, the generator behaves as if seeded by Seed(1)." wheee!!!!